package.json

```diff
@@ -20,14 +20,26 @@
     "Henrik Vendelbo"
   ],
   "dependencies": {
+    "connect-static": "latest",
+    "serve-static": "latest",
+    "serve-favicon": "latest",
+    "express-session": "latest",
+    "keygrip": "latest",
+    "cookies": "latest",
+    "csurf": "latest",
+    "body-parser": "latest",
+    "cookie-session": "latest",
+    "compression": "latest",
+    "cookie-parser": "latest",
+    "jwt-simple":"latest",
     "apitree": "= 1.2.0",
     "chokidar": "= 0.12.2",
     "clean-css": "2.2.22",
     "colors": "= 1.0.3",
     "commander": "2.5.0",
-    "connect": "2.27.6",
```

Comment:
Upgrading connect and engine.io is great, but shouldn't it be a separate feature branch ?